### PR TITLE
Troubleshoot ReadTheDocs

### PR DESCRIPTION
**Pull request by jtpio**
## References

Just trying to troubleshoot https://github.com/jupyterlite/jupyterlite/issues/963.

## User-facing changes

None

## Backwards-incompatible changes

None

### CODE REVIEW

File: dodo.py [hunk 1]
```diff
@@ -647,12 +647,13 @@ def _optimize_images():
                 cwd=str(P.ROOT),
             )
 
-        yield dict(
-            name="post:images",
-            doc="clean sphinx-jsonschema duplicate ids",
-            actions=[_optimize_images],
-            file_dep=[B.YARN_INTEGRITY],
-        )
+        if not C.RTD:
```
Comment (jtpio) [inline dodo.py:650]: Probably doesn't make sense to skip on RTD since the task is about optimizing svgs in the docs folder.

**Comment by jtpio** [comment]
So when trying in a fresh env there should be some svg files around:

image

Not sure why they are not found on RTD.

**Comment by bollwyvl** [comment]
Maybe rtd changed their build location, and it needs to be passed in or set via env var?

**Comment by jobovy** [comment]
> Maybe rtd changed their build location, and it needs to be passed in or set via env var?

Yes, they changed the build location to `_readthedocs`: readthedocs/readthedocs.org/pull/9888.

**Comment by bollwyvl** [comment]
This would probably work until there's a documented env var:

```py
# conf.py
def after_build(app: Sphinx, error):
    """sphinx-jsonschema makes duplicate ids. clean them"""
    os.environ.update("JLITE_DOCS_OUT", app.builder.outdir)  # <--- dodo.py already looking for this
    do_tasks("post", RTD_POST_TASKS)
```

**Comment by bollwyvl** [comment]
Ah, maybe one is coming:

https://github.com/readthedocs/readthedocs.org/pull/9913

But the above is probably still the right play for now.

**Comment by bollwyvl** [comment]
sry, of course my syntax bad:

```py
os.environ.update(JLITE_DOCS_OUT=app.builder.outdir) 
```

**Comment by bollwyvl** [comment]
yay docs

**Pull request merged**